chat-plugins/wifi.js

```diff
@@ -224,9 +224,10 @@ var LotteryGiveAway = (function () {
 		for (var id in this.winners) {
 			finallist.push(this.winners[id].name);
 		}
+		var finallength = finallist.length;
```

Comment:
Yeah, I suppose I could do that. So it'd be something like-
var finallength = (finallist.length === 1 ? 'winner' : 'winners');